.github/workflows/turing.yaml

```diff
@@ -383,20 +388,33 @@ jobs:
         working-directory: sdk
         run: make setup
 
-      - name: Run End-to-End Test Suite with Turing API
+      # - name: Run End-to-End Test Suite with Turing API
```

Comment:
I saw UPI e2e tests using the current structure, but this PR converts test to use ginkgo. Hence, I placed ginkgo tests in `test-e2e` folder to differentiate, and we can decide what to do before this PR is merged. Any thoughts? @krithika369 @pradithya

Reply:
Since the UPI tests are in a branch at the moment, IMO we can go ahead and make the necessary changes for the test framework replacement in this PR and get it merged to `main` first. Then, when the changes from `main` are pulled back to `grpc_upi`, the UPI test cases must be adapted - this task can lie on either team depending on the bandwidth. @pradithya / @leonlnj, do you need us to pick this up once this PR is merged?

Reply:
oh weird i didnt get any notification, just happen to stumble on this. 

Im not too sure on the urgency of other tasks, let me check with Aria

Reply:
I didn't realize this change.
Can you merge it to `main`? I'll try to adapt upi e2e test afterwards but timebox it to only today. I'll let you know if I need your help.

Reply:
Sure thing @pradithya ! Happy to chime in!